Assets/Scripts/Runes/RuneCount.cs

```diff
@@ -162,6 +162,23 @@ public void RemovedARune(Rune.RuneType type)
     			SelectFire();
     		}
     	}
+
+    	// If didn't pick up in placement mode, show the UI briefly
+    	if (rp.canPlace != true)
+    	{
+    		StartCoroutine(FlashUI());
+    	}
+    }
+
+    // Coroutine to turn on the UI, then off
+    IEnumerator FlashUI()
```

Comment:
How much of a "flash" is this? I don't think it would be cause for concern with like epilepsy, but he might point it out if it's potentially harmful.

Reply:
Not at all a flash, probably a misleading name. Really just means the rune placement icons in the left turn on for a few seconds, then back off